cmd/kubeadm/app/util/staticpod/utils.go

```diff
@@ -220,6 +220,9 @@ func WriteStaticPodToDisk(componentName, manifestDir string, pod v1.Pod) error {
 
 // ReadStaticPodFromDisk reads a static pod file from disk
 func ReadStaticPodFromDisk(manifestPath string) (*v1.Pod, error) {
+	if _, err := os.Stat(manifestPath); os.IsNotExist(err) {
+		return nil, err
+	}
```

Comment:
this is part of the general logic used elsewhere. would it cause other unexpected repercussions?
can we instead just leave it unchanged and handle it on the side of callers of ReadStaticPodFromDisk for the reset command?

Reply:
+1